### DISCUSSION TITLE
Add a support to change start menu action

### DISCUSSION THREAD

**Discussion by flourigh** [valinet/ExplorerPatcher#2922] [Ideas]
In default, start button action open a start menu, but, a option to change command like Open Shell

![image](https://github.com/valinet/ExplorerPatcher/assets/722723/8a8f8cd4-8c66-4429-9b0b-f3d4e42a0289)

to another personalized command is much powerful

**Comment by haydengriz** [comment]
I think this might be added, I do know that they are eventually adding a windows 10 styled 7 button
